src/main/java/pl/wrapper/parking/infrastructure/inMemory/InMemoryRepository.java

```diff
@@ -0,0 +1,10 @@
+package pl.wrapper.parking.infrastructure.inMemory;
+
+import java.io.Serializable;
+import java.util.Set;
+
+public interface InMemoryRepository<K extends Serializable, V extends Serializable> {
+    void add(K key, V value);
+    V get(K key);
+    Set<K> fetch();
```

Comment:
1. Change method name to fetchAllKeys().
2. Add a method fetchAllEntries that returns a a Set of Entries.